packages/blocks/src/multiple-choice-answer/index.js

```diff
@@ -46,7 +48,12 @@ const MultipleChoiceAnswer = ( { attributes, className } ) => {
 			<CheckboxAnswer
 				attributes={ attributes }
 				className={ classes }
-				inputProps={ inputProps }
+				isMultiSelect={ isMultiSelect }
+				isSelected={ isSelected }
+				onChange={ ( event ) =>
+					onUpdate( event.target.value, isSelected )
```

Comment:
As you guessed, my idea here was to avoid repetition.
But I agree, maybe it's too much for now.
If it starts to hurt us, we can think about the helpers.